src/main/java/car/controller/RacingGameController.java

```diff
@@ -0,0 +1,54 @@
+package car.controller;
+
+import car.model.*;
+import car.view.Constant;
+import car.view.InputView;
+import car.view.OutputView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGameController {
+
+    private final InputView input = new InputView();
+    private final CarName carName = new CarName();
+    private RacingCars racingCars;
+    private GameNumber gameNumber;
+    private final OutputView output = new OutputView();
+    private final Winner winner = new Winner();
```

Comment:
사소하지만 비슷한 것끼리 정리되면 보기 좋은 것 같아요!
```suggestion
    private final InputView input = new InputView();
    private final CarName carName = new CarName();
    private final OutputView output = new OutputView();
    private final Winner winner = new Winner();
    private RacingCars racingCars;
    private GameNumber gameNumber;
```